### DISCUSSION TITLE
Is there any similar UI/UX open-source project for creating a ChatGPT like web-app interface?

### DISCUSSION THREAD

**Discussion by sundar7D0** [lencx/ChatGPT#800] [General]
Is there any similar UI/UX open-source project for creating a ChatGPT like web-app interface?

**Comment by raulmarindev** [comment]
> Is there any similar UI/UX open-source project for creating a ChatGPT like web-app interface?

Do you mean like these?

https://github.com/Yidadaa/ChatGPT-Next-Web
https://github.com/mckaywrigley/chatbot-ui
https://github.com/chathub-dev/chathub

**Comment by sundar7D0** [comment]
Yep. These UI/UX are really great as many users got acquainted with the ChatGPT UI/UX now, it would be easier to bank on what worked well! Thanks
